indico/modules/events/contributions/lists.py

```diff
@@ -122,7 +122,7 @@ def render_list(self, contrib=None):
         :param contrib: Used in RHs responsible for CRUD operations on a
                         contribution.
         :return: dict containing the list's entries, the fragment of
-                 displayed entries and whether the contrib passed is displayed
+                 displayed entries and whether the contribution passed is displayed
```

Comment:
Docstrings should wrap after 80 characters and python code after 120. Make sure you setup your editor to indicate that 😉